lib/bolt_server/transport_app.rb

```diff
@@ -182,6 +196,15 @@ def run_script(target, body)
       [@executor.run_script(target, file_location, body['arguments'])]
     end
 
+    def apply_catalog(target, body)
+      # TODO: sync plugins to env cahce, tar them up, base-64 encode and pass to task
+      body['apply_options'] ||= {}
+      body['action'] = 'apply'
+      results = @executor.run_task(target, @apply_task, body)
```

Comment:
I think we will only ever have a single target at a time?

Reply:
Yep, only ever one target in these requests